staging/src/k8s.io/csi-translation-lib/translate_test.go

```diff
@@ -20,7 +20,19 @@ import (
 	"reflect"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/csi-translation-lib/plugins"
+)
+
+var (
+	defaultZoneLabels = map[string]string{
+		v1.LabelZoneFailureDomain: "us-east-1",
```

Comment:
The values are flipped. Region should be "us-east-1"

Reply:
Fixed